api/roleAssignment/roleAssignmentService.ts

```diff
@@ -0,0 +1,15 @@
+import { NextFunction, Response } from 'express';
+import { EnhancedRequest } from '../lib/models';
+
+export async function getJudicialRoles(req: EnhancedRequest, res: Response, next: NextFunction): Promise<any> {
+  const personRoles = [
+    {roleId: 'lead', roleName: 'Lead judge'},
+    {roleId: 'hearing', roleName: 'Hearing judge'}];
+  return res.send(personRoles).status(200);
+}
+
+export async function getLegalOpsRoles(req: EnhancedRequest, res: Response, next: NextFunction): Promise<any> {
+  const personRoles = [
+    {roleId: 'manager', roleName: 'Case manager'}];
+  return res.send(personRoles).status(200);
+}
```

Comment:
```suggestion
export async function getTypeOfRoles(req: EnhancedRequest, res: Response, next: NextFunction): Promise<any> {
  const personRoles = [
    {roleId: 'leadJudge', roleName: 'Lead judge', roleType: 'judicial'},
    {roleId: 'hearingJudge', roleName: 'Hearing judge', roleType: 'judicial'},
    {roleId: 'caseManager', roleName: 'Case manager', roleType: 'legalOps'}];
  return res.send(personRoles).status(200);
}
```
You could merge this data in node layer, on Angular layer you don't need to call two APIs and join them, but just filter the typeOfRole by roleType.

Reply:
This is a good point. Was going to say I wasn't sure how the data would be returned from the API but it doesn't matter because we can merge the data within this function